app/javascript/app/service-provider-form.js

```diff
@@ -10,53 +10,36 @@ $(function(){
 
   var toggle_form_fields = function(id_protocol){
     switch(id_protocol){
-      case 'openid_connect':
+      case 'openid_connect_private_key_jwt':
         $('.saml-fields').hide();
         $('.oidc-fields').show();
+        $('.cert-fields').show();
+        break;
+      case 'openid_connect_pkce':
+        $('.saml-fields').hide();
+        $('.oidc-fields').show();
+        $('.cert-fields').hide();
```

Comment:
if an SP that uses PKCE ever wants to submit RISC events to our backend, they'll need a cert...is there a way to show the certs but mark them as optional?

Reply:
Whoa good catch, did not realize that. I'll add some text to that effect to the field label. Thanks!